### PR TITLE
Added Support for EC2 Images (AMIs)

### PR DESCRIPTION
**Pull request by kedarghule**
Alright, I went to resolve the conflicts and it merged those cloudanix files in the ec2-enhancements branch as well 🙈

### CODE REVIEW

**Comment by kedarghule** [comment]
I have created a new branch called aws-ec2-enhancements which is like a "fixed" version of this. So I'll open a PR from that and close this PR then.

**Comment by kedarghule** [comment]
I have created a new branch called aws-ec2-enhancements which is like a "fixed" version of this. So, you review and merge this PR. Whatever changes you suggest here, I'll do them in that branch as well and then once this PR is merged, I'll delete this branch.

File: cartography/data/indexes.cypher [hunk 1]
```diff
@@ -42,6 +42,7 @@ CREATE INDEX ON :DOProject(id);
 CREATE INDEX ON :DynamoDBGlobalSecondaryIndex(id);
 CREATE INDEX ON :DynamoDBTable(arn);
 CREATE INDEX ON :DynamoDBTable(id);
+CREATE INDEX ON :EC2Images(id);
```
Comment (mpurusottamc) [inline cartography/data/indexes.cypher:45]: We should rename this to EC2Image without the `s`

Comment (mpurusottamc) [reply cartography/data/indexes.cypher:45]: Make necessary adjustments in other files accordingly

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
minor changes needed

File: cartography/data/indexes.cypher (hunk 1 above)
Comment (kedarghule) [reply cartography/data/indexes.cypher]: Done

**Review by mpurusottamc — APPROVED** [review]
Thanks

**Pull request merged**